pkg/report/output/output.go

```diff
@@ -16,23 +17,52 @@ import (
 	"github.com/bearer/curio/pkg/types"
 	"gopkg.in/yaml.v3"
 
+	"github.com/hhatto/gocloc"
 	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 )
 
 var ErrUndefinedFormat = errors.New("undefined output format")
 
-func ReportPolicies(report types.Report, output *zerolog.Event, config settings.Config) (bool, error) {
+func ReportPolicies(report types.Report, output *zerolog.Event, config settings.Config) (reportPassed bool, err error) {
+	reportSupported := false
+	reportPassed = false
+	err = nil
+
+	lineOfCodeOutput, err := stats.GoclocDetectorOutput(config.Scan.Target)
+	if err != nil {
+		return
+	}
+
+	reportSupported, err = anySupportedLanguagesPresent(lineOfCodeOutput , config)
+	if err != nil {
+		return
+	}
+
+	if !reportSupported {
+		placeholderStr, err2 := getPlaceholderOutput(report, config, lineOfCodeOutput)
+		if err2 != nil {
+			err = err2
+			return
+		}
```

Comment:
Minor, but would something like this work? 

```suggestion
		var placeholderStr *strings.Builder
		placeholderStr, err = getPlaceholderOutput(report, config, lineOfCodeOutput)
		if err != nil {
			return
		}
```

Reply:
That does work, thanks!